src/components/quick-delegate-banner/quick-delegate-banner.tsx

```diff
@@ -48,27 +50,28 @@ export const QuickDelegateBannerComponent = (
             const clientMinAmount = await blockchainConfig
                 .getClient(props.chainId)
                 .getMinimumAmountDelegate();
-
             const gasLimit = blockchainConfig.config.feeOptions.defaults.gasLimit[tokenConfig.type];
             const gasPrice = blockchainConfig.config.feeOptions.defaults.gasPrice;
             const fees = gasPrice.multipliedBy(gasLimit).toFixed();
 
             const minAmount = new BigNumber(clientMinAmount).plus(new BigNumber(fees));
-
             // getMinimumAmountDelegate + fees + 1%
             setMinimumAmountDelegate(minAmount.plus(minAmount.dividedBy(100)));
         })();
     }, [props.blockchain, props.account.address, props.chainId]);
 
     if (tokenConfig.ui.tokenScreenComponent === TokenScreenComponentType.DELEGATE) {
+        const avaialableChartData = props.accountStats?.chartStats.find(
```

Comment:
don't we have this value in redux ? I'm not happy with this solution, if we will add internationalisation `item.title` will not be `Available`. This data should be use only for display purposes.